test-cypress/cypress/commands/api/user.d.ts

```diff
@@ -0,0 +1,26 @@
+// load the global Cypress types
+/// <reference types="cypress" />
+
+declare namespace Cypress {
+    interface Chainable {
+        /**
+         * create user and save api credentials further use
+        */
+        apiCreateUser(username: string, log: boolean): Chainable<Element>
```

Comment:
All of these custom commands are declared as returning `Chainable<Element>` - which I think is the correct convention for Cypress custome commands - but your actual commands don't return the cy.request which would actually make it chainable.